### DISCUSSION TITLE
❗ Add Callout Cards to Orbit

### DISCUSSION THREAD

**Discussion by alexasselin008** [gsoft-inc/sg-orbit#1119] [New components Ideas]
## Description

We have a component that encourages users to take an action related to a new feature or opportunity

![image](https://user-images.githubusercontent.com/38871812/194599879-3ccfa685-ecd7-475c-9592-dc1768982c09.png)
![image](https://user-images.githubusercontent.com/38871812/194599924-a85e12ec-56ad-480c-828e-2bf1e8445332.png)

**Comment by alexasselin008** [comment]
References: 
https://polaris.shopify.com/components/callout-card

**Comment by patricklafrance** [comment]
I suggest moving this to a Github discussion.
